java/client/src/test/java/glide/api/RedisClusterClientTest.java

```diff
@@ -24,7 +25,7 @@ public void custom_command_returns_single_value() {
 
         var client = new TestClient(commandManager, "TEST");
 
-        var value = client.customCommand(new String[0]).get();
+        var value = client.customCommand(new String[] {"test"}).get();
```

Comment:
why is the added argument needed?

Reply:
Well, because the server will fail on the backend if you pass an empty list.  So this better represents our use case... even if our white box testing doesn't case.